src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -29,13 +29,27 @@
 matrix of order `n` exists if and only if `n= 1, 2` or `n` is a multiple
 of `4`.
 
-The module below implements the Paley constructions (see for example
-[Hora]_) and the Sylvester construction. It also allows you to pull a
-Hadamard matrix from the database at [SloaHada]_.
+The module below implements constructions of Hadamard and skew Hadamard matrices
+for all known orders `\le 1000`, plus some more greater than `1000`. It also
+allows you to pull a Hadamard matrix from the database at [SloaHada]_.
+
+    sage: unknown_hadamard = [668, 716, 892]
+    sage: unknown_skew_hadamard = [356, 404, 428, 476, 596, 612, 668, 708, 712, 716,
+    ....:                         764, 772, 804, 808, 820, 836, 856, 892, 900, 916,
+    ....:                         932, 940, 952, 980, 996]
+    sage: for n in range(1, 251): # not tested
```

Comment:
```suggestion
allows you to pull a Hadamard matrix from the database at [SloaHada]_.

The following code will test this by replacing `k = 250`.
This can take around 10 minutes, so we check up to `\le 200`::

    sage: k = 50
    sage: unknown_hadamard = [668, 716, 892]
    sage: unknown_skew_hadamard = [356, 404, 428, 476, 596, 612, 668, 708, 712, 716,
    ....:                         764, 772, 804, 808, 820, 836, 856, 892, 900, 916,
    ....:                         932, 940, 952, 980, 996]
    sage: for n in range(1, k+1):
```
Alternatively
```suggestion
allows you to pull a Hadamard matrix from the database at [SloaHada]_.

The following verifies this, but it can take around 10 minutes::

    sage: unknown_hadamard = [668, 716, 892]
    sage: unknown_skew_hadamard = [356, 404, 428, 476, 596, 612, 668, 708, 712, 716,
    ....:                         764, 772, 804, 808, 820, 836, 856, 892, 900, 916,
    ....:                         932, 940, 952, 980, 996]
    sage: for n in range(1, 251):  # not tested
```
I would prefer the first onesince it makes sure some part of the code is  tested.